app/Controllers/feedController.php

```diff
@@ -755,8 +755,9 @@ public function contentSelectorPreviewAction() {
 
 		//Get first entry (syntax robust for Generator or Array)
 		foreach ($entries as $myEntry) {
-			$entry = $myEntry;
-			break;
```

Comment:
We must not break because we need to consume the result of the buffered query fully before starting a new one.